guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -1,31 +1,21 @@
 [id="Building_a_Discovery_Image_{context}"]
-= Building a Discovery image
+= Building a custom Discovery image
 
-The Discovery image is a minimal operating system that is PXE-booted on hosts to acquire initial hardware information and to check in with {Project}.
-Discovered hosts keep running the Discovery image until they are rebooted into Anaconda, which then initiates the provisioning process.
+You can build a custom Discovery ISO or rebuild an ISO if you change configuration files.
 
-ifdef::satellite[]
-The operating system image is based on {RHEL} 7.
-endif::[]
-ifndef::satellite[]
-Run the following procedure on {EL} 8.
+The Discovery image uses a minimal operating system that is booted on hosts to acquire initial hardware information and check in with {Project}.
+Discovered hosts keep running the Discovery operating system until they are rebooted into Anaconda, which then initiates the provisioning process.
 
-The Discovery image is based on CentOS Stream 8.
-endif::[]
+[IMPORTANT]
+====
+Do not use your production {ProjectServer} nor {SmartProxyServer} to perform this procedure.
+Use either a dedicated environment or copy the repositories and Kickstart file to a separate server.
+====
 
-ifdef::satellite[]
-The `{fdi-package-name}` package contains this image.
-You must install the package on the {SmartProxy} that provides TFTP services.
-endif::[]
+The following procedure demonstrates the building process on {EL} 8 and uses CentOS Stream 8 repositories.
 
-Use this procedure to build a {Project} discovery image or rebuild an image if you change configuration files.
-
-Do not use this procedure on your production {Project} or {SmartProxy}.
-Use either a dedicated environment or copy the synchronized repositories and a kickstart file to a separate server.
-
-ifndef::satellite[]
 .Prerequisites
-* Ensure that hardware virtualization is available on your host.
+* Ensure that hardware virtualization is available on your server.
```

Comment:
I know that this is outside the scope of your changes but prerequisites should not be mini procedures. No need to tell the user how to use 'dnf install' or 'git clone'. If you can, please change them to bullet list:

* Install the following packages:
** git-core
** lorax
** etc.

* Clone the `foreman-discovery-image` repository.

Reply:
Not needed, but convenient. The user can just copy the commands to ensure those things.
Since this procedure isn't part of Satellite, I'm leaving it as it is.